src/solc_0.6/polygon/SandPolygonDepositor.sol

```diff
@@ -0,0 +1,27 @@
+//SPDX-License-Identifier: MIT
+// solhint-disable-next-line compiler-version
+pragma solidity 0.6.6;
+import "./ERC20Extended.sol";
+import "@maticnetwork/pos-portal/contracts/root/RootChainManager/IRootChainManager.sol";
```

Comment:
I see, the reason we need 0.6.6 is because of this
Would be better to make a PR on their side to make version more flexible. Interfaces should be easy to make more generic version

Reply:
But for now you think we should keep it that way ? should their fix block this PR ?

Reply:
I think in the mean time, a better approach is to use our own interface file, just copy from @maticnetwork/pos-portal/contracts/root/RootChainManager/IRootChainManager.sol